packages/wrangler/src/utils/print-bindings.ts

```diff
@@ -565,20 +572,43 @@ export function printBindings(
 			" (".length +
 			maxValueLength +
 			")".length;
+
+		const shouldWrap =
+			bindingLength + 6 + maxTypeLength + 6 + maxModeLength >=
+			process.stdout.columns;
+
 		logger.log(title);
-		logger.log(
-			`${padEndAnsi(dim("Binding"), bindingLength)}      ${padEndAnsi(dim("Resource"), maxTypeLength)}      ${hasMode ? dim("Mode") : ""}`
-		);
-		for (const binding of output) {
-			const bindingString = padEndAnsi(
-				`${white(`env.${binding.name}`)}${binding.value ? ` (${dim(typeof binding.value === "symbol" ? chalk.italic("inherited") : binding.value ?? "")})` : ""}`,
-				bindingLength
+		if (shouldWrap) {
+			logger.log(
+				`${padEndAnsi(
+					dim("Binding"),
+					bindingPrefix.length + maxNameLength
+				)}    ${padEndAnsi(dim("Resource"), maxTypeLength)}    ${hasMode ? dim("Mode") : ""}`
 			);
-
+		} else {
 			logger.log(
-				`${bindingString}      ${brandColor(binding.type.padEnd(maxTypeLength))}      ${hasMode ? binding.mode : ""}`
+				`${padEndAnsi(dim("Binding"), bindingLength)}      ${padEndAnsi(dim("Resource"), maxTypeLength)}      ${hasMode ? dim("Mode") : ""}`
```

Comment:
As far as I can tell, the only difference between these two blocks is `bindingPrefix.length + maxNameLength` vs `bindingLength`. Is that right?

If so then perhaps simplify by creating a new variable for this value and only having one log statement?